### PR TITLE
Add CSSAngleValue and a test

### PR DESCRIPTION
**Pull request by wilddamon**
Spec: https://drafts.css-houdini.org/css-typed-om/#anglevalue-objects

### CODE REVIEW

File: src/css-angle-value.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+(function(internal, scope) {
+
+  AngleType = {
+    DEG: "deg",
+    RAD: "rad",
+    GRAD: "grad",
+    TURN: "turn"
+  };
+
+  function isValidAngleUnit(unit) {
```
Comment (rjwright) [inline src/css-angle-value.js]: This is analogous to CSSLengthValue.isValidLengthType. Should make terminology consistent.

File: src/css-angle-value.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    return internal.objects.any(AngleType, function(type) {
+      return unit == type;
+    });
+  }
+
+  function rad2deg(rad) {
```
Comment (rjwright) [inline src/css-angle-value.js]: I would prefer to call these radToDeg etc. unless there is some precedent for this naming that I don't know about.

File: src/css-angle-value.js (hunk 1 above)
Comment (wilddamon) [inline src/css-angle-value.js]: Changed to isValidAngleType. Kept it as a private function for now.

File: src/css-angle-value.js (hunk 2 above)
Comment (wilddamon) [inline src/css-angle-value.js]: Done. These are what the internal methods in blink are called. No need to follow that though.

**Comment by rjwright** [comment]
lgtm

**Pull request merged**